models/demos/t3000/falcon40b/tt/falcon_attention.py

```diff
@@ -338,16 +336,12 @@ def fwd_prefill(
         # K Cache update
         ttnn.fill_cache(
             layer_past[0],
-            ttnn.experimental.typecast(
-                key_layer, self.model_config["KV_CACHE_DTYPE"], memory_config=ttnn.DRAM_MEMORY_CONFIG
-            ),
+            ttnn.typecast(key_layer, self.model_config["KV_CACHE_DTYPE"], memory_config=ttnn.DRAM_MEMORY_CONFIG),
             user_id,
         )
         ttnn.fill_cache(
             layer_past[1],
-            ttnn.experimental.typecast(
-                value_layer, self.model_config["KV_CACHE_DTYPE"], memory_config=ttnn.DRAM_MEMORY_CONFIG
-            ),
+            ttnn.typecast(value_layer, self.model_config["KV_CACHE_DTYPE"], memory_config=ttnn.DRAM_MEMORY_CONFIG),
```

Comment:
this can to stay ttnn.experimental.typecast. it refers to typecast in ttnn/operations/experimental/copy